### ISSUE TITLE
Torque run between nodes

### ISSUE THREAD

**Issue by sbenkorichi** [firemodels/fds#6059]
Glenn,

I'm setting some nodes, running jobs on them from headnode works fine. But, the moment, when I try to use 2 or more together, it just freezes, qstat -a shows job is running, .err and .log files got output but no fds files, when I kill the job, I receive this: 
![image](https://user-images.githubusercontent.com/20113030/36591929-895d23ae-188b-11e8-94ff-3aab1aeeaf42.png)

I wonder why it get stuck at the background when it reaches mpiexec line, it can't output stuff after mpiexec line. Any thoughts on this? Thanks !

**Comment by gforney** [comment]
have no idea.  
1. make sure your ssh keys are setup ie you can slogin to each node without a password
2. make sure that the directory containing mpiexec, fds etc or whatever you are running is cross mounted on each node

**Comment by sbenkorichi** [comment]
This one is just never experienced. I'm setting up M1000e, and this probably has to do with the way the blades are communicating to each other, as I do have two types of ethernet (one reserve it for ethernet/internet) and the other one is set as static ip to communicate between blades (with 10GB speed). 

I can run single jobs up to 12 MPI processes, with any blade running it from the headnode, but I can't use two blades at same time.  When, I use two blades (excluding headnode), immediately, the job get cancelled, although in pbsnodes, all of them are free and up. But, when including the headnode with another blade, I receive the above error. 

I will keep digging.

**Comment by gforney** [comment]
you can use 
tracejob id
to get a clue for what went wrong - where id is the number reported by qstat -a .  when you type pbsnodes make sure the names of the nodes reported correspond to the 10 gig interfaces

**Comment by sbenkorichi** [comment]
I believe this is solved now.
What I found out, is that I had to provide the ips of each computing node on all of them so they can communicate with each other when the. 
Your point was right, the job getting cancelled was due to the ssh, I also found that stack wasn't set to unlimited although it I already set it in .bashrc and .profile, I had to reset it again on the `/etc/security/limits.conf `
as : 
```
*    soft   stack     unlimited 
*    hard   stack     unlimited 
```
The surprising thing is that I also had to add the root as well. 
```
root    soft   stack     unlimited 
root    hard   stack     unlimited 
```
Also, I found I was making a silly error of exporting home directory into other nodes, which caused this confusion of ssh, as it fakes it, that it already exists, 
phew !
